services/common/src/redux/slices/permitServiceSlice.ts

```diff
@@ -34,6 +34,7 @@ const permitExtractionStatusMap = {
 export interface PermitExtraction {
   task_status: PermitExtractionStatus;
   task_id: string;
+  permit_amendment_document_guid: string;
```

Comment:
this bit makes a TS error on line 148 (where it fetches task status). Should the permit_amendment_document_guid be optional? Either that or it wants to be set to a value on that line (which probably doesn't make sense)